### PR TITLE
Added Post Verification Page

### PR DESCRIPTION
**Pull request by hacker7599**
**Created Routes for verification

### CODE REVIEW

**Review by jaydeepukani — APPROVED** [review]

File: app/Http/Controllers/User/PostScheduler/SchedulerController.php [hunk 1]
```diff
@@ -194,4 +194,10 @@ private static function validateUploadMedia($fileInfo, $platforms)
             }
         }
     }
+
+    public function verificaition()
+    {
+        return view('user.post.verification');
+    }
+
```
Comment (jaydeepukani) [inline app/Http/Controllers/User/PostScheduler/SchedulerController.php:202]: Remove this if not needed

File: resources/views/user/post/scheduling.blade.php [hunk 2]
```diff
@@ -267,4 +268,38 @@
...
+                                    <th scope="col" rowspan="2" data-sortable="true" data-field="supported_type">Supported Media Type</th>
+                                    <th scope="col" colspan="2">Max Size Supported</th>
+                                </tr>
+                                <tr>
+                                    <th scope="col" data-sortable="true" data-field="images_size">Image</th>
+                                    <th scope="col" data-sortable="true" data-field="video_size">Video</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                            </tbody>
```
Comment (jaydeepukani) [inline resources/views/user/post/scheduling.blade.php:298]: Why body is empty?

File: app/Http/Controllers/User/PostScheduler/SchedulerController.php (hunk 1 above)
Comment (hacker7599) [reply app/Http/Controllers/User/PostScheduler/SchedulerController.php:202]: Removed

File: resources/views/user/post/scheduling.blade.php (hunk 2 above)
Comment (hacker7599) [reply resources/views/user/post/scheduling.blade.php:298]: Spandan Has to give that to me so initially he was going to do it now I'll do it on priority

File: app/Http/Controllers/User/PostScheduler/SchedulerController.php (hunk 1 above)
Comment (jaydeepukani) [reply app/Http/Controllers/User/PostScheduler/SchedulerController.php:202]: But the code is still present!

File: resources/views/user/post/scheduling.blade.php (hunk 2 above)
Comment (jaydeepukani) [reply resources/views/user/post/scheduling.blade.php:298]: Okay

**Comment by hacker7599** [comment]
Solved

File: app/Http/Controllers/User/PostScheduler/SchedulerController.php (hunk 1 above)
Comment (hacker7599) [reply app/Http/Controllers/User/PostScheduler/SchedulerController.php]: Check Now

File: resources/views/user/post/scheduling.blade.php (hunk 2 above)
Comment (hacker7599) [reply resources/views/user/post/scheduling.blade.php]: Please Check Now I have pushed the changes

**Review by jaydeepukani — APPROVED** [review]

**Pull request merged**